models/intel/smartlab-sequence-modelling-0001/README.md

```diff
@@ -0,0 +1,80 @@
+# online mstcn++
+
+## Use Case and High-Level Description
+This is an online action segmentation network for 16 classes trained on Intel dataset. It is an online version of MSTCN++. The difference between online MSTCN++ and MSTCN++ is that the former accept stream video as input while the latter assume the whole video is given.
```

Comment:
why do you duplicate changes from https://github.com/openvinotoolkit/open_model_zoo/pull/3039 ?